src/config/permissions.js

```diff
@@ -0,0 +1,22 @@
+import db from '../models';
+
+const { Permission } = db;
```

Comment:
you still have a db query here. Please reach out to your colleagues to see how you can remove query

Reply:
Ok